.github/fabricbot.json

```diff
@@ -0,0 +1,6871 @@
+            "cston"
+          ]
+        },
+        {
+          "labels": [
+            "area-Infrastructure-libraries"
+          ],
+          "mentionees": [
+            "safern"
+          ]
+        },
+        {
+          "labels": [
+            "area-Infrastructure"
+          ],
+          "mentionees": [
+            "dotnet/runtime-infrastructure"
+          ]
+        },
+        {
+          "labels": [
+            "area-GC-coreclr"
+          ],
+          "mentionees": [
+            "dotnet/gc"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Net"
+          ],
+          "mentionees": [
+            "dotnet/ncl"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Net.Http"
+          ],
+          "mentionees": [
+            "dotnet/ncl"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Net.Security"
+          ],
+          "mentionees": [
+            "dotnet/ncl",
+            "vcsjones"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Net.Sockets"
+          ],
+          "mentionees": [
+            "dotnet/ncl"
+          ]
+        },
+        {
+          "labels": [
+            "area-Diagnostics-coreclr"
+          ],
+          "mentionees": [
+            "tommcdon"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Diagnostics"
+          ],
+          "mentionees": [
+            "tommcdon"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Data"
+          ],
+          "mentionees": [
+            "roji",
+            "ajcvickers"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Data.OleDB"
+          ],
+          "mentionees": [
+            "roji",
+            "ajcvickers"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Data.Odbc"
+          ],
+          "mentionees": [
+            "roji",
+            "ajcvickers"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Data.SqlClient"
+          ],
+          "mentionees": [
+            "cheenamalhotra",
+            "david-engel"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.ComponentModel.DataAnnotations"
+          ],
+          "mentionees": [
+            "ajcvickers",
+            "bricelam",
+            "roji"
+          ]
+        },
+        {
+          "labels": [
+            "area-Extensions-FileSystem"
+          ],
+          "mentionees": [
+            "maryamariyan",
+            "dotnet/area-extensions-filesystem"
+          ]
+        },
+        {
+          "labels": [
+            "area-Extensions-HttpClientFactory"
+          ],
+          "mentionees": [
+            "dotnet/ncl"
+          ]
+        },
+        {
+          "labels": [
+            "area-System.Net.Quic"
```

Comment:
Nit: Not alphabetically sorted next to other area-System.Net.*

Reply:
This file is the verbatim contents of the exported configuration from the Fabric Bot portal. I thought I'd avoid making changes during the migration to Config-As-Code. I'm planning on iterating with improvements once migration is complete.